src/iconsManifest/supportedExtensions.ts

```diff
@@ -1193,8 +1193,23 @@ export const extensions: IFileCollection = {
     },
     {
       icon: 'cypress',
-      extensions: [`cypress.json`, `cypress.env.json`],
+      extensions: [
+        `cypress.json`,
+        `cypress.env.json`,
+        `cypress.config.js`,
+        `cypress.config.ts`,
+        `cypress.config.cjs`,
+        `cypress.config.mjs`,
```

Comment:
Really nitpicky, but why are you using template literals here?
```suggestion
        'cypress.json',
        'cypress.env.json',
        'cypress.config.js',
        'cypress.config.ts',
        'cypress.config.cjs',
        'cypress.config.mjs',
```

Reply:
wasn't on purpose, thanks